tests/t288_arg_oct.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+
+import re
+
+from runtest import TestBase
+
+class TestCase(TestBase):
+    def __init__(self):
+        TestBase.__init__(self, 'arg_oct', result="""
+# DURATION     TID     FUNCTION
+   0.560 us [1587104] | __monstartup();
+   0.170 us [1587104] | __cxa_atexit();
+            [1587104] | main() {
+ 120.272 us [1587104] |   creat("bar.foo", 0755) = 4;
+   3.210 us [1587104] |   chmod("bar.foo", 0777) = 0;
+   6.350 us [1587104] |   unlink("bar.foo") = 0;
+ 131.752 us [1587104] | } = 0; /* main */
+""", cflags='-g')
```

Comment:
The `cflags='-g'` is not needed because this tests only for known library calls, which are described at `misc/prototypes.h`.
```suggestion
""")
```